src/coreclr/src/ToolBox/superpmi/superpmi-shared/standardpch.h

```diff
@@ -3,6 +3,9 @@
 // Licensed under the MIT license. See LICENSE file in the project root for full license information.
 //
 
+#ifndef __STANDARDPCH_H__
```

Comment:
Can you please drop the `__` at the beginning and end to reduce the debt we have with using identifier names that should be internal for compiler use?